cardano_node_tests/utils/tx_view.py

```diff
@@ -183,6 +210,17 @@ def check_tx_view(  # noqa: C901
     if tx_raw_len_certs != loaded_len_certs:
         raise AssertionError(f"certificates: {tx_raw_len_certs} != {loaded_len_certs}")
 
+    for certificate in tx_loaded.get("certificates") or []:
+        certificate_name = list(certificate.keys())[0]
+        certificate_fields = set(list(certificate.values())[0].keys())
+
+        if CERTIFICATES_INFORMATION.get(certificate_name) and not certificate_fields.issubset(
+            CERTIFICATES_INFORMATION[certificate_name]
+        ):
+            raise AssertionError(
+                f"The output of the certificate {certificate_name} does not has the expected fields"
```

Comment:
Optional: "does not have" and put `{certificate_name}` to single quotes for better readability.